src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -66,6 +68,7 @@ TriggeredSubscription::TriggeredSubscription
   format            = _format;
   reference         = _reference;
   attrL             = _attrL;
+  cacheSubReference = NULL;
```

Comment:
I understand that you have added a new fiel to the TriggeredSubscription class in order to store a reference to the cache. I undesratnd that is is NULL for csubs find in the DB directly (i.e. the ones without patterns).

However, do we really need this taking into account the existence of methods to search by ID?